### PR TITLE
chore(plugins): codegen from wrapscan

### PR DESCRIPTION
**Pull request by cbrzn**

### CODE REVIEW

File: packages/plugins/ethereum-wallet/polywrap.yaml [hunk 1]
```diff
@@ -4,4 +4,4 @@ project:
   name: ethereum-wallet
 source:
   module: ./Cargo.toml
-  schema: ./polywrap.graphql
\ No newline at end of file
+  schema: ./src/polywrap.graphql
```
Comment (nerfZael) [inline packages/plugins/ethereum-wallet/polywrap.yaml:7]: What's the reason for this change? If I'm not mistaken, we've been using the schema in the root (instead of src) more recently. This also IMO, makes more sense since it's where the manifest is and schema (like the manifest) is not a source file. It's also less likely to make IDEs complain

Comment (cbrzn) [reply packages/plugins/ethereum-wallet/polywrap.yaml:7]: no strong reason - actually we were not using it and i just updated the path to make use of it; can def move it to root folder

**Review by nerfZael — APPROVED** [review]
